src/common/store/action-utils.ts

```diff
@@ -0,0 +1,25 @@
+import { markInThunkActionWithTabId, retrieveActionTab } from './tabs/action-tab';
+
+/*
+ * The primary actions (ordinary objects {} ) are marked by webext-redux with _sender.
+ * This allows to establish the tab for which the action was issued in the reducer
+ * Thunk actions, however, are functions issuing more functions within their body.
```

Comment:
Some missing/redundant words in the sentence make it hard to get the sense of it.